plugin-modernizer-cli/src/test/java/io/jenkins/tools/pluginmodernizer/cli/MavenInvokerIntegrationTest.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.tools.pluginmodernizer.cli;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.apache.maven.shared.invoker.*;
+import org.junit.jupiter.api.Test;
+import java.io.File;
+import java.util.Arrays;
+
+public class MavenInvokerIntegrationTest {
+
+    @Test
+    public void testMavenGoalExecution() throws MavenInvocationException {
+        InvocationRequest request = new DefaultInvocationRequest();
+        request.setPomFile(new File("pom.xml"));
+        request.setGoals(Arrays.asList("clean", "verify"));
```

Comment:
We should run not run clean verify on the root pom

But invoke something like `mvn exec:exec -Dexec.executable="java"`.....` by running the CLI previously build and packaged in a previous phase